test/test_callables.py

```diff
@@ -1423,13 +1423,38 @@ def test_inline_stride():
                     dtype=np.float64,
                     shape=("n", "n")),
                 ...],
-            assumptions="n>=1",
+            assumptions="n>=2",
```

Comment:
Why this change?

Reply:
The test is wrong. Previously, the loopy array bounds checker was not smart enough to give an error about needing `n>=2`, but now it is, because the array accesses are now affine compared to the previous non-affine expressions like `i//n`.